stac_fastapi/elasticsearch/tests/api/test_api.py

```diff
@@ -66,20 +65,17 @@ def test_transactions_router(api_client):
 
 
 @pytest.mark.skip(reason="unknown")
-def test_app_transaction_extension(app_client, load_test_data, es_transactions):
+def test_app_transaction_extension(app_client, load_test_data, es_txn_client):
     item = load_test_data("test_item.json")
     resp = app_client.post(f"/collections/{item['collection']}/items", json=item)
     assert resp.status_code == 200
-    time.sleep(1)
-    es_transactions.delete_item(
+
+    es_txn_client.delete_item(
         item["id"], item["collection"], request=MockStarletteRequest
     )
 
 
-def test_app_search_response(load_test_data, app_client, es_transactions):
-    item = load_test_data("test_item.json")
-    es_transactions.create_item(item, request=MockStarletteRequest)
-
+def test_app_search_response(app_client, ctx):
```

Comment:
the `ctx` fixture clears the indexes, loads an item, runs the tests, and then clears the indexes again